Userland/Libraries/LibGfx/ICC/TagTypes.cpp

```diff
@@ -760,7 +760,7 @@ ErrorOr<NonnullRefPtr<NamedColor2TagData>> NamedColor2TagData::from_bytes(Readon
 
     auto& header = *bit_cast<NamedColorHeader const*>(bytes.data() + 8);
 
-    unsigned const record_byte_size = 32 + sizeof(u16) * (3 + header.number_of_device_coordinates_of_each_named_color);
+    u64 const record_byte_size = 32 + sizeof(u16) * (3 + header.number_of_device_coordinates_of_each_named_color);
```

Comment:
Aren't we now potentially overflowing on the next line with `2 * sizeof(u32) + sizeof(NamedColorHeader) + header.count_of_named_colors * record_byte_size`?

Reply:
Good point, I've modified it so that we can't overflow there either.